webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -24,6 +24,9 @@ const initialState = {
         issuetype: {
             name: '',
         },
+        priority: {
```

Comment:
No need for a default here. There are lots of possible values and we can't default them all.